met-web/src/components/common/index.tsx

```diff
@@ -16,6 +17,10 @@ import LoadingButton from '@mui/lab/LoadingButton';
 import { MET_Header_Font_Family, MET_Header_Font_Weight } from '../../styles/constants';
 import { When } from 'react-if';
 
+export const MobileToolbar = styled(Toolbar)(() => ({
```

Comment:
I forgot why we're using a toolbar for spacing instead of just adding some margin. This a bit confusing to any maintainer. Could we find a better way to have that spacing. A <span> with padding would at least make more sense if there isn't any other idea